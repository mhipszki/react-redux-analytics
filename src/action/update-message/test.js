import updateMessage from './';

test('generates action with given message', () => {
  expect(updateMessage('message')).toMatchSnapshot();
});
