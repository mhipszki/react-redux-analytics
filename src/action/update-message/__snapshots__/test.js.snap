// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates action with given message 1`] = `
Object {
  "message": "message",
  "type": "UPDATE_MESSAGE",
}
`;
